### ISSUE TITLE
Error- / Warning- / Info-Message Handling

### ISSUE THREAD

**Issue by manuelmeister** [ecamp/ecamp3#1277]
Features:
- [x] #3037
- [x] Translatable messages (#2902)
- [x] Source component decides if it handles the message itself or if the global handler gets used
- [ ] #3076

**Comment by usu** [comment]
@BacLuc Are you working on _"Global component to display the messages"_ or is this free to work on?

**Comment by BacLuc** [comment]
I did not yet start with the global component to display messages.
Maybe we have merge conflicts then if we both work on serverError.js, but we can resolve them.
If you want to start with that, feel free.

**Comment by usu** [comment]
I gave a shot to include [vue-toastification](https://github.com/Maronato/vue-toastification). Looks neat in my opinion and brings all the features we'd need to have. They have a version for Vue2 and one for Vue3. Decent amount of usage/stars/forks. Not a lot of activity recently, though.

I struggled to get it working inside ecamp3, however. Not sure where the issue is. Messages get added to the proper component, but aren't displayed. I would have said it's very similar to the issue described in https://github.com/Maronato/vue-toastification/issues/247#issue-941989113 (`getPositionToasts` not being a computed). But then again, the demo is working and others seem to use it without issue. So maybe an issue from my end.

If someone want to give it a try, feel free.

**Other options:**
- Use a another package that is based on vuetify snackbar, e.g. https://github.com/eolant/vuetify-toast-snackbar or https://github.com/jaredhan418/vuetify-toast-snackbar-ng. Both of them are quite old and have little traction, so not sure if we really want to depend on them.
- Implement something on our own: For sure an option. Just a bit a pity, as this seems like a problem that could be solved generically.
